src/routes/Game.vue

```diff
@@ -432,10 +430,16 @@ export default {
       this.updatePlay(data);
       logEvent("game_exited", data);
     },
-    async gameEnded() {
+    async gameEnded(isGameOver) {
       this.instance.destroyInstance();
       this.isGameEnded = true;
       let achievementPromise = Promise.resolve();
+      if (isGameOver === true) {
```

Comment:
```suggestion
      if (isGameOver) {
```

**Unless** you are trying to coerce `isGameOver` from something non-boolean to a `boolean` type, which I don't think is allowed by strict `===` equality, don't think `=== true` is necessary. There are linter rules which can be setup to catch redundant comparisons like these ;)

Reply:
Actually, I think because I called the function somewhere as a callback from an audio library, it actually returns something else in the param, thus the need to check for boolean value here.